persistence-jdbc/core/src/main/scala/com/lightbend/lagom/internal/persistence/jdbc/SlickProvider.scala

```diff
@@ -34,8 +33,8 @@ private[lagom] class SlickProvider(system: ActorSystem)(implicit ec: ExecutionCo
 
   val autoCreateTables: Boolean = createTables.getBoolean("auto")
 
-  val db = SlickDatabase.forConfig(readSideConfig, slickConfig)
-  val profile = SlickDriver.forDriverName(readSideConfig)
+  val db = SlickUtils.forConfig(readSideConfig, slickConfig)
+  val profile = SlickUtils.forDriverName(readSideConfig)
```

Comment:
If this is the only code that uses these methods, maybe they should be inlined here rather than introducing a new singleton object.

Reply:
On second thought, that would further complicate the copyright situation, so let's not :smile: